pages/common/ollama.md

```diff
@@ -19,6 +19,14 @@
 
 `ollama list`
 
+- Update pulled models (Referenced from <https://github.com/ollama/ollama/issues/1890#issuecomment-1885381713>):
+
+`ollama list | tail -n +2 | awk '{print $1}' | while read -r model; do  sudo ollama pull $model; done`
+
+- Update Ollama:
```

Comment:
```suggestion
- Upgrade Ollama on Linux:
```
In Windows and Mac, the updates are installed automatically and you are prompted to apply them unlike Linux. Checkout https://github.com/ollama/ollama/blob/main/docs/faq.md#how-can-i-upgrade-ollama to know more.

Reply:
Thanks for your suggestion. Yes you're correct, I thought `tldr` is mainly used by linux users, so I didn't put much care about other OS users. Thanks for your suggestions!

Reply:
And you're right, it's upgrade not update! Should I amend the commit message?